File: stress/t/arr_fillsum.c
#include <stdio.h>
#include <stdint.h>
int main(void){
  int64_t i,s,t,n;
  int64_t A[50];
  n=50;
  for(i=0;i<n;i++){ A[i]=i*3-7; }
  s=0;
  for(i=0;i<n;i++){ t=A[i]; s=s+t; }
  printf("sum="); printf("%ld",(long)s); printf("\n");
  t=A[25];
  printf("a25="); printf("%ld",(long)t); printf("\n");
  return 0;
}

File: stress/t/arr_fillsum.f
      PROGRAM ARRFILLSUM
      INTEGER*8 I,S,T,N
      INTEGER*8 A(0:49)
      N=50
      I=0
   10 IF (I.GE.N) GOTO 20
      A(I)=I*3-7
      I=I+1
      GOTO 10
   20 CONTINUE
      S=0
      I=0
   30 IF (I.GE.N) GOTO 40
      T=A(I)
      S=S+T
      I=I+1
      GOTO 30
   40 CONTINUE
      WRITE(*,'(A,I0)') 'sum=',S
      T=A(25)
      WRITE(*,'(A,I0)') 'a25=',T
      END
